cleaning_services/demo/hr_department.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="hr.dep_administration" model="hr.department" forcecreate="1">
+    <field name="name">Administration</field>
+  </record>
+  <record id="hr_department_2" model="hr.department" forcecreate="1">
```

Comment:
`forcecreate` is only useful for records coming from another module. Please remove it for all records defined here (I know it comes from base_module, but no excuse :smile: )
```suggestion
  <record id="hr_department_2" model="hr.department">
```

Reply:
up